### PR TITLE
Add minVersion and targetVersion to Realm Transformer metrics

### PR DESCRIPTION
**Pull request by bmeike**
Fixes https://github.com/realm/realm-java-private/issues/206

### CODE REVIEW

File: realm-transformer/src/main/groovy/io/realm/transformer/RealmTransformer.groovy [hunk 1]
```diff
@@ -174,11 +174,14 @@ class RealmTransformer extends Transform {
             it.getPackageName()
         }
 
+        def targetSdk = project?.android?.defaultConfig?.targetSdkVersion?.mApiLevel as String;
+        def minSdk = project?.android?.defaultConfig?.minSdkVersion?.mApiLevel as String;
```
Comment (zaki50) [inline realm-transformer/src/main/groovy/io/realm/transformer/RealmTransformer.groovy]: `targetSdkVersion` and `minSdkVersion` depend on product flavor. To collect more precise information,  we should obtain them from the product flavor currently building.

I'm looking for a way to determine the product flavor.

Comment (bmeike) [inline realm-transformer/src/main/groovy/io/realm/transformer/RealmTransformer.groovy]: Well, only if they are changed by the flavor.  I believe all flavors inherit from the defaultConfig, yes?

Comment (zaki50) [inline realm-transformer/src/main/groovy/io/realm/transformer/RealmTransformer.groovy]: yes, but each flavor can override those versiin values.

Comment (bmeike) [inline realm-transformer/src/main/groovy/io/realm/transformer/RealmTransformer.groovy]: @zaki50 : Looks like pretty much a hack:  http://stackoverflow.com/questions/30621183/how-to-get-current-flavor-in-gradle

**Pull request merged**